pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -29,6 +29,13 @@ import (
 	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
+const (
+	hypervIsolationAnnotationKey = "windows.kubernetes.io/isolation-type"
```

Comment:
Please include `experimental` in the key. I want to make it clear that this is not an official API, and won't be supported long term -- eventually we will have a native secure container concept that this should move to.

Reply:
ack